helm-chart/README.md

```diff
@@ -74,7 +74,9 @@ helm install --name my-release -f myvalues.yaml .
 | `service.loadBalancer.enabled`    | If enabled will create an LB for the connector, ClusterIP otherwise | `true`     |
 | `service.loadBalancer.annotations`| Annotations to set to the LB service        | `service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "4000"` |
 | `dropChunk.schedule`              | The schedule with which the drop-chunk Job runs | `0,30 * * * *`                 |
+| `dropChunk.successLimit`          | The number of successful dropChunk pods to retain in-cluster | `5`               |
```

Comment:
I think I'd prefer `dropChunk.successfulJobsHistoryLimit` and `dropChunk.failedJobsHistoryLimit` because `successLimit` sounds like it will stop running the job after X successes. This is really about the history.